vm/src/builtins/pytype.rs

```diff
@@ -467,6 +467,10 @@ impl PyType {
             }
         }
 
+        // TODO: how do we know if it should have a dict?
```

Comment:
```suggestion
```

Reply:
ah, missed it! (btw for small changes like these, feel free to commit the change directly)